### PR TITLE
Add note on CLI authentication

### PR DESCRIPTION
**Pull request by dlevy-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -201,6 +201,16 @@ For further information on usage:
 Azure Active Directory authentication uses temporary authentication tokens to authenticate.
 The `mssql` package does not provide an implementation to obtain tokens: instead, import the `azuread` package and use driver name `azuresql`. This driver uses [azidentity](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#section-readme) to acquire tokens using a variety of credential types.
 
+To reduce friction in local development, `DefaultAzureCredential` can authenticate as the user signed into the Azure CLI.
```
Comment (shueybubbles) [inline README.md]: `DefaultAzureCredential` isn't the string we use in the connection string.
An app would use either `ActiveDirectoryDefault` or `ActiveDirectoryAzCli` for the `fedauth` connection string parameter.

Comment (shueybubbles) [reply README.md]: this example probably belongs after line 221

**Review by shueybubbles — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dlevy-msft) [reply README.md]: I thought about pushing it down to the end but I thought it was important to include it in the overview section to give people a fast path for local development to get started quickly.

File: README.md [hunk 2]
```diff
@@ -201,6 +201,16 @@ For further information on usage:
 Azure Active Directory authentication uses temporary authentication tokens to authenticate.
 The `mssql` package does not provide an implementation to obtain tokens: instead, import the `azuread` package and use driver name `azuresql`. This driver uses [azidentity](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity#section-readme) to acquire tokens using a variety of credential types.
 
+To reduce friction in local development, `ActiveDirectoryDefault` can authenticate as the user signed into the Azure CLI.
+
+Run the following command to sign into the Azure CLI before running your application using the `ActiveDirectoryDefault` connection string parameter:
```
Comment (shueybubbles) [inline README.md:206]: >`ActiveDirectoryDefault` c [](http://example.com/codeflow?start=95&length=26)

``fedauth=ActiveDirectoryDefault`` or ``fedauth=ActiveDirectoryAzCli``

**Review by shueybubbles — APPROVED** [review]
:shipit:

**Pull request merged**